src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.Scanner;
+import java.util.ArrayList; 
+import java.text.ParseException;
+
 public class Duke {
-    public static void main(String[] args) {
+    void logo(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+    static void greeting(){
+        String output = "    ____________________________________________________________\n" +
+        "     Hello! I'm Duke\n" + 
+        "     What can I do for you?\n" +
+        "    ____________________________________________________________\n";
+        System.out.println(output);
+    }
+    static String addDoubleLine(String str){
+        String line = "    ____________________________________________________________";
+        return line + "\n" + str + "\n" + line;
+    }
+    static void handleList(ArrayList<Task> ls){
```

Comment:
It would be better if you can give a more descriptive name for function parameter(s). You may refer to Java coding standards about the proper way of doing it.